src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -15,7 +15,7 @@
 
 from adal import AdalError
 from azure.mgmt.resource.subscriptions.models import \
-    (SubscriptionState, Subscription, SubscriptionPolicies, SpendingLimit)
+    (SubscriptionState, Subscription, SubscriptionPolicies, SpendingLimit, ManagedByTenant)
```

Comment:
Because subscription models have api contraint, please use get_models() here.

Reply:
This test is only for the latest. For specific test, it is at `test_profile_v2016_06_01.py`.

Reply:
This test is only for `latest`. For specific test, it is in `test_profile_v2016_06_01.py`.